pylint/testutils/configuration_test.py

```diff
@@ -18,6 +20,11 @@
 PylintConfiguration = Dict[str, ConfigurationValue]
 
 
+if not PY38_PLUS:
+    # See https://stackoverflow.com/a/56935186
+    copy._deepcopy_dispatch[type(re.compile(""))] = lambda r, _: r  # type: ignore[attr-defined]
```

Comment:
Is this relevant for this change? I don't see `copy` being used here?

Reply:
Yes, we're starting to deepcopy a regex pattern in the functional configuration test which does not work in python 3.6 or 3.7.

Reply:
Could you add a comment as to why this is needed? Because it is 25 lines detached from where we need this this isn't immediately clear.